src/dos/program_imgmount.cpp

```diff
@@ -36,12 +36,56 @@
 #include "string_utils.h"
 #include "../ints/int10.h"
 
+void IMGMOUNT::ListImgMounts(void)
+{
+	const std::string header_drive = MSG_Get("PROGRAM_MOUNT_STATUS_DRIVE");
+	const std::string header_name = MSG_Get("PROGRAM_MOUNT_STATUS_NAME");
+	const std::string header_label = MSG_Get("PROGRAM_MOUNT_STATUS_LABEL");
+	const std::string header_slot = MSG_Get("PROGRAM_MOUNT_STATUS_SLOT");
+
+	const int term_width = real_readw(BIOSMEM_SEG, BIOSMEM_NB_COLS);
+	const auto width_1 = static_cast<int>(header_drive.size());
+	const auto width_3 = std::max(11, static_cast<int>(header_label.size()));
+	const auto width_4 = std::max(11, static_cast<int>(header_slot.size()));
+	const auto width_2 = term_width - 4 - width_1 - width_3 - width_4;
+
+	auto print_row = [&](const std::string &txt_1, const std::string &txt_2,
+	                     const std::string &txt_3, const std::string &txt_4) {
+		WriteOut("%-*s %-*s %-*s %-*s\n", width_1, txt_1.c_str(),
+		         width_2, txt_2.c_str(), width_3, txt_3.c_str(),
+		         width_4, txt_4.c_str());
```

Comment:
Is the `11` related to `"%-*s %-*s %-*s %-*s\n"`? 
Maybe mention something about the 11 being the minimum allowable width(if that's what it is).

Reply:
The 11 is based on the MOUNT code, and now I define it as a const int named "minimum_length" which is common to both IMGMOUNT and MOUNT commands.